great_expectations/core/expectation_configuration.py

```diff
@@ -149,792 +149,6 @@ class ExpectationConfiguration(SerializableDictDot):
         ValueError: If a `domain_type` cannot be determined.
     """
 
-    kwarg_lookup_dict: ClassVar[Mapping[str, KWargDetailsDict]] = {
```

Comment:
If we are getting rid of this we can probably delete the `KWargDetailsDict` class as well.

Reply:
There's still one other use of it. I think that can go as well, but it effects a number of other methods in this file, so I might keep it for a followup PR.